### DISCUSSION TITLE
WinMergeU - CLI - Is there any parameter or way by which winmerge closes after generating the output file?

### DISCUSSION THREAD

**Discussion by pcakhilnadh** [WinMerge/winmerge#741] [Q&A] [answered]
Hi There,

I am trying to integrate WinMerge application with my python program.

```

cli = r' "C:\Program Files (x86)\WinMerge\WinMergeU.exe" somepath/file1.txt  + somepath/file2.txt -or output.html -s -minimize'

os.system("cmd /k " +cli)

print("OK")
```
the WinMerge Application opens up and compares the documents and generates the output html to a specific location mentioned in the cli but the control is not going back to python to print OK. If I close WinMerge the Python program terminates without printing OK.

Is there any parameter or way by which WinMerge closes after generating the output file?

Thank you.

**Comment by sdottaka** [comment, marked as answer]
Try adding a `-noninteractive` switch to `cli`

**Reply by pcakhilnadh** [reply]
That was perfect. Thank you.
